governance/votes/Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,202 @@
+- The current Output Conduit will be deauthorized.
+- The old `RWA015_A_OUTPUT_CONDUIT_LEGACY` will be removed from the chainlog as it will no longer be used.
+
+### Smart Burn Engine Deployment
+
+As per this successful govenrance [poll](https://vote.makerdao.com/polling/QmQmxEZp) and the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#9-surplus-buffer-and-smart-burn-engine) a new Smart Burn Engine will be deployed along with the following parameters, if this executive proposal passes.
+
+- Deploy the new Smart Burn Engine as `MCD_FLAP`.
+- The [Maximum System Surplus (`vow.hump`)](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-auction-limit) will be reduced by 200 million DAI from 250 million DAI to **50 million DAI**.
+- The [Surplus Auction Lot Size (`vow.bump`)](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-lot-size) will be reduced by 25,000 DAI from 30,000 DAI to **5,000 DAI**.
+- The `hop` will be set to **1,577 seconds**. The `hop` parameter defines the minimum amount of time that must elapse between `kick`s, therefore setting the maximum frequency of market actions.
+- The `want` will be set to **0.98**. The `want` parameter defines the relationship between the purchase price of MKR and the price provided by the MKR/USD oracle, i.e. the slippage. A want of 0.98 means that the purchase price of MKR can be up to 2% worse than the oracle price. However, if the price of MKR on Uniswap is better than the Oracle price, the Smart Burn Engine will not be constrained.
+- The `pip` will be set to `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df`. The `pip` is the oracle referenced by the Smart Burn Engine.
+- The `receiver` will be set to the Pause Proxy address - `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df `. The `receiver` is the address that LP tokens will be deposited to after each market action.
+- A `FlapperJob` will be added to the chainlog and keeper network sequencer. This will allow the Maker Keeper Network to trigger the Smart Burn Engine when able to do so.
+
+The new Smart Burn Engine will purchase MKR tokens for Dai from the Surplus Buffer. This MKR will then be paired with more Dai from the Surplus Buffer and deposited to UniswapV2's DAI/MKR pool. The LP tokens will be stored in the Pause Proxy.
+
+As there can only be one `Flapper` active in the Maker Protocol, the old `MCD_FLAP` used for buy-and-burn will be deactivated.
+
+Please review the [discussion thread](https://forum.makerdao.com/t/introduction-of-smart-burn-engine-and-initial-parameters/21201) for more information.
+
+### Keeper Job Updates
+
+As per this [post](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292) from the Protocol Scope Facilitators, the following addresses related to the Maker Keeper Network (DssCron) will be added to the [Chainlog](https://chainlog.makerdao.com/), if this executive proposal passes.
+
+- **Sequencer:** `0x238b4E35dAed6100C6162fAE4510261f88996EC9`
```

Comment:
In general, add Etherscan links or no?

Reply:
typically haven't done this for these kind of entries, it's different to e.g. a payment address. See recent example here: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20June%2014%2C%202023.md#relevant-smart-contract-information